bindings/pydeck/examples/gpu_grid_layer.py

```diff
@@ -0,0 +1,30 @@
+import pydeck as pdk
+import pandas as pd
+
+GPUGRID_LAYER_DATA = ('https://raw.githubusercontent.com/uber-common/'
+                      'deck.gl-data/master/website/sf-bike-parking.json')
+df = pd.read_json(GPUGRID_LAYER_DATA)
+
+# Define a layer to display on a map
+layer = pdk.Layer(
+    'GPUGridLayer',
+    df,
+    pickable=True,
+    extruded=True,
+    cellSize=200,
+    elevation_scale=4,
+    get_position='COORDINATES',
+)
+
+# Set the viewport location
+view_state = pdk.ViewState(
+    longitude=-122,
+    latitude=37,
+    zoom=6,
+    min_zoom=5,
+    max_zoom=15,
+    pitch=20.5,
+    bearing=-15.36)
```

Comment:
Nit: Use identical formatting for Layer and ViewState - which line last parenthesis goes on, whether last param has comma etc. Not sure if there are such conventions in Python or other examples to follow?

Reply:
Using Black to format the Python should solve this: https://github.com/psf/black